aten/src/ATen/native/mps/operations/Normalization.mm

```diff
@@ -718,10 +718,13 @@ static string get_mem_string(c10::MemoryFormat memory_format) {
                                                                   secondaryTensor:epsilonTensor
                                                                              name:nil];
 #ifdef __MAC_15_0
-          rsqrtTensor = [mpsGraph reciprocalSquareRootWithTensor:varianceEpsTensor name:nil];
-#else
-          rsqrtTensor = [mpsGraph reverseSquareRootWithTensor:varianceEpsTensor name:nil];
-#endif
+          if (@available(macOS 15.0, *)) {
```

Comment:
> do not work

Can you elaborate? I wrote this diff because I was building on macOS 14 with a macOS-15-capable SDK, and tests were failing because reciprocalSquareRootWithTensor:name: was not available at runtime. This diff fixed that, so I'm unclear on how it doesn't work.